.dev.devcontainer.pmm-managed.yaml.example

```diff
@@ -0,0 +1,81 @@
+services:
```

Comment:
why do we need one more configuration file, even two?

Reply:
I run pmm-managed in docker container (devcontainer) - `.dev.devcontainer.pmm-managed.yaml`
and locally - `.dev.pmm-managed.yaml`

these are example configs

Reply:
having multiple environments is not a good practice. We created devcontainer to have the same environment for all of us.

Reply:
okay, I will remove it from the PR and leave it for my personal productivity